### PR TITLE
Tooltips for CSV import and import page refactoring

### PR DESCRIPTION
**Pull request by RumovZ**
As always, please go ahead if you see any room for improvement, especially with the fluent strings or the styling, which I'm not super confident with.
@abdnh, your input is welcome as well, of course!

### CODE REVIEW

File: ftl/core/importing.ftl [hunk 1]
```diff
@@ -188,6 +188,36 @@ importing-note-updated-as-file-had-newer = Note updated, as file had newer versi
 importing-note-skipped-due-to-missing-notetype = Note skipped, as its notetype was missing
 importing-note-skipped-due-to-missing-deck = Note skipped, as its deck was missing
 importing-note-skipped-due-to-empty-first-field = Note skipped, as its first field is empty
+importing-field-separator-help =
+    The character delimiting fields in the text file. You can use the preview to check
```
Comment (dae) [inline ftl/core/importing.ftl:192]:
```suggestion
    The character separating fields in the text file. You can use the preview to check
```

(easier for the layperson)

File: ftl/core/importing.ftl [hunk 2]
```diff
@@ -188,6 +188,36 @@ importing-note-updated-as-file-had-newer = Note updated, as file had newer versi
...
+    if the file is currently rendered into columns correctly.
```
Comment (dae) [inline ftl/core/importing.ftl:193]:
```suggestion
    if the fields are separated correctly.
```

File: ftl/core/importing.ftl [hunk 3]
```diff
@@ -188,6 +188,36 @@ importing-note-updated-as-file-had-newer = Note updated, as file had newer versi
...
+    
+    Please note that if this character appears in any field itself, the field has to be
+    quoted accordingly to the CSV standard. Spreadsheet programs like LibreOffice will
+    do this automatically.
+importing-allow-html-in-fields-help =
+    Enable this if the file contains HTML formatting. E.g. if the file contains the string
+    '&lt;br&gt;', it will appear as a line break on your card. On the other hand, with this
+    option disabled, the literal characters '&lt;br&gt;' will be rendered.
+importing-notetype-help =
+    Newly imported notes will have this notetype, and only existing notes with this
```
Comment (dae) [inline ftl/core/importing.ftl:203]:
```suggestion
    Newly-imported notes will have this notetype, and only existing notes with this
```

File: ftl/core/importing.ftl [hunk 4]
```diff
@@ -188,6 +188,36 @@ importing-note-updated-as-file-had-newer = Note updated, as file had newer versi
...
+    What to do if an imported note matches an existing one.
+    
+    - `{ importing-update }`: Update the existing note.
+    - `{ importing-preserve }`: Do nothing.
+    - `{ importing-duplicate }`: Create a new note.
+importing-match-scope-help =
+    Only existing notes with the same notetype will be checked for duplicates. This can
+    additionally be restricted to notes with cards in the same deck.
+importing-tag-all-notes-help =
+    These tags will be added to both newly imported and updated notes.
```
Comment (dae) [inline ftl/core/importing.ftl:219]:
```suggestion
    These tags will be added to both newly-imported and updated notes.
```

**Review by dae — COMMENTED** [review]
Will try to review this properly tomorrow, but one issue I noticed so far: when importing an .apkg file, progress is no longer shown correctly.

**Comment by RumovZ** [comment]
Thanks. Should've tested with larger imports.

**Review by dae — COMMENTED** [review]
Thanks Rumo! This is a nice improvement. And no problem, that's what review is for :-)

**Pull request merged**